src/components/TextField/TextField.tsx

```diff
@@ -24,11 +25,21 @@ export interface TextFieldProps extends BaseTextFieldProps, DebouncedOnChangePro
   underlined?: boolean;
 }
 
+interface Focusable {
+  /**
+   * Sets focus to the input, and places cursor at the end of the value.
+   * @public
+   */
+  focus(): void;
+}
```

Comment:
This was a shot at surfacing this function in the stlyeguidist docs, but didn't work.  Any ideas?

Reply:
I got the idea from here: https://react-styleguidist.js.org/docs/documenting.html#public-methods

Reply:
Maybe it's because of https://github.com/styleguidist/react-docgen-typescript.  I don't see anything about functions there.